### DISCUSSION TITLE
Which protocols support Smart Synchronization?

### DISCUSSION THREAD

**Discussion by StraightfaceStudios** [iterate-ch/cyberduck#14362] [Q&A] [answered]
I know SMB on windows isn't supported.  But which protocols do support Smart Synchronization? Does SMB on MacOS allow smart synchronization?  WebDav?  

What would be the best way to allow WAN clients to use smart sync with a Windows File Server while allowing LAN clients to access the SMB share directly?

**Comment by dkocher** [comment, marked as answer]
All [protocols](https://docs.mountainduck.io/protocols/) available in Mountain Duck support [_Smart Synchronization_](https://docs.mountainduck.io/mountainduck/sync/).

**Reply by AlanATL** [reply]
I'd like to rephrase the OP's question by asking: Is it (or will it soon be) possible for Mountain Duck to connect directly to an SMB share in order to sync with a Windows File Server? I have this need as well. Currently, I am looking into another alternative called OwlFiles, but I would much rather use Mountain Duck for this if possible.

**Reply by dkocher** [reply]
This [^1]  is currently being worked on.

[^1]: #5368

**Reply by AlanATL** [reply]
Excellent, looking forward to it!
